go.mod

```diff
@@ -1,3 +1,17 @@
+// NB: Since this repostiory was scaffolded using `ignite s chain pocket`
+// but the module was later rename to github.com/pokt-network/poktroll, we
+// need to swap the two lines below only during the scaffolding operation.
+// NOTE that the operation is expected to error out but all the necesary
+// types and functions will be scaffolded correctly.
+// Ref: https://github.com/ignite/cli/issues/3737
+//
+// The following will need to be done manually after the scaffold:
```

Comment:
This seems like a good time for a bash script IMO

Reply:
I would like to propose that we entertain the idea of preferring to write scripts in go. I think the stdlib is more than capable on it's own to accomplish most scripting needs. While it may be verbose in some cases, I think minimizing the number of languages in the repo is worth it on the whole.

Reply:
@Pantani responded to it, so I want to see if we can solve it via the ignite path first before investing time into it writing proprietary scripts.